dev/check-license

```diff
@@ -35,7 +35,7 @@ acquire_rat_jar () {
       wget --quiet ${URL} -O "$JAR_DL" && mv "$JAR_DL" "$JAR"
     else
       printf "You do not have curl or wget installed, please install rat manually.\n"
-      exit -1
+      exit 255
```

Comment:
SC2242 (error): Can only exit with status 0-255. Other data should be written to stdout/stderr.